server/routes/check_verification.js

```diff
@@ -43,7 +48,7 @@ module.exports = function(req,res,constants,helpers,Sentry,client,base,airtableR
                     },
                     sentry_extras, Sentry
                   );
-                  res.status(400).send(reason1);
+                  res.status(500).send(reason1);
```

Comment:
500 errors are usually server errors. I would suggest using something in the 400's

Reply:
I kept the errors which are due to failed information fetch from the APIs we depend on - Airtable, Google Place details and Twilio as 500

Reply:
Should I change to 400 something?

Reply:
On some research, 500s do not make sense in this case, I will change them all to 403(Forbidden)